drivers/interrupt_controller/loapic_intr.c

```diff
@@ -93,8 +94,24 @@ static int loapic_init(struct device *unused)
 	ARG_UNUSED(unused);
 	s32_t loApicMaxLvt; /* local APIC Max LVT */
 
-	/* enable the Local APIC */
-	x86_write_loapic(LOAPIC_SVR, x86_read_loapic(LOAPIC_SVR) | LOAPIC_ENABLE);
+	/*
+	 * enable the local APIC. note that we use xAPIC mode here, since
```

Comment:
indent...

Reply:
@nashif Not sure what you mean ...

Reply:
ignore, my browser is showing wrong indentation.